database/sqlite/sqlite_aclk_alert.c

```diff
@@ -939,18 +939,14 @@ void aclk_push_alert_snapshot_event(char *node_id __maybe_unused)
 #endif
 }
 
-#define SQL_DELETE_ALERT_ENTRIES "DELETE FROM aclk_alert_%s WHERE filtered_alert_unique_id NOT IN (SELECT unique_id FROM health_log_%s);"
-
+#define SQL_DELETE_ALERT_ENTRIES "DELETE FROM aclk_alert_%s WHERE filtered_alert_unique_id + 604800 < UNIXEPOCH();" //TODO check the when from health log
```

Comment:
The `UNIXEPOCH()` returns current time, so unless I am missing something, I understand that you want to delete on `filtered_alert_unique_id  < UNIXEPOCH() - 604800`, please, correct me if I am wrong.

Reply:
Yes, we want to delete entries that are 1 week old from there. The filtered_alert_unique_id is a timestamp (although not an exact one, but close enough on a weekly scale). I will change to use the when from the actual alert which should be more exact.

Reply:
Please, also add `MAX_REMOVED_PERIOD` to the query.

Reply:
All right @MrZammler , the current query was removing one week ahead.